packages/mobile/src/components/lineup-tile/LineupTile.tsx

```diff
@@ -64,10 +68,10 @@ export const LineupTile = ({
   const isOwner = user_id === currentUserId
   const isCollection = 'playlist_id' in item
   const isTrack = 'track_id' in item
-  const trackId = isTrack ? item.track_id : undefined
-  const streamConditions = isTrack ? item.stream_conditions : null
+  const contentId = isTrack ? item.track_id : item.playlist_id
+  const streamConditions = item.stream_conditions ?? null
```

Comment:
cool that this can be consolidated!

Reply:
Yeah! The mobile components were far easier to work with than web. It was pretty much already set for this feature minus a few conditionals.